requirements/base.txt

```diff
@@ -1,32 +1,32 @@
 commitment<3.0.0
-dj-database-url==0.5.0
-django-apiblueprint-view==2.3.2
+dj-database-url==1.0.0
+django-apiblueprint-view
 django-basicauth==0.5.3
-django-extensions==3.1.5
+django-extensions==3.2.1
 django-localflavor==3.1
-django-markdown-deux==1.0.5
-django==3.2.13
+markdown==3.3.7
```

Comment:
I made this change bc `django-markdown-deux==1.0.5` was no longer being maintained and was not compatible with `Django>4.0`. As I'm making the same change to WCIVF, I'm only now realising that I'm not sure this package is actually being used.

Reply:
AFAIK There are 2 places it is used in WDIV - for rendering the explainers and cancelled election notifications in
https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/polling_stations/templates/election_explainers.html
and
https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/polling_stations/templates/fragments/cancelled_election.html
but it looks like you've caught them both in this PR anyway.